src/garage/tf/algos/_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Utility functions used by tensorflow algorithms."""
+
+
+class _Default:  # pylint: disable=too-few-public-methods
+    """A wrapper class to represent default arguments.
+
+    Args:
+        val (object): Argument value.
+
+    """
+
+    def __init__(self, val):
+        self.val = val
+
+
+def make_optimizer(optimizer_type, **kwargs):
+    """Create an optimizer for tensorflow algos.
+
+    Args:
+        optimizer_type (type): Type of optimizer.
```

Comment:
`optimizer_type` can be either a `type`, or `tuple[type, dict]`. This selection can be expressed by `Union[type, tuple[type, dict]]`